src/Containers/FeaturesProfileCard/FeaturedProfilesCard.tsx

```diff
@@ -20,10 +20,13 @@ export interface IProfile {
 
 export interface IFeaturedProfilesCardProps {
     profileData: IProfile[];
+    counterBool: boolean;
```

Comment:
Documentation of Props goes here not on the component. You are doucmenting the type